virt/virt-4-10-release-notes.adoc

```diff
@@ -145,39 +156,37 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
-//Known issues at time of 4.9 release
-//BZ-2007397
-* If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, you might lose data. This is due to a race condition that can cause the VM disk's contents to be wiped from the persistent volume. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])
+* When you create a VM, the VM's unique fully qualified domain name (FQDN) now contains the VM's cluster domain name. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1998300[*BZ#1998300*])
 
-// Review this for permanent doc home in 4.10
-* Editing a virtual machine fails if the VM references a deleted template that was provided by {VirtProductName} before version 4.8. In {VirtProductName} 4.8 and later, deleted {VirtProductName}-provided templates are automatically recreated by the {VirtProductName} Operator.
+* If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, then you no longer lose data. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])
 
-* If a cloning operation is initiated before the source is available to be cloned, the operation stalls indefinitely. This is because the clone authorization expires before the cloning operation starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855182[*BZ#1855182*])
-+
-** As a workaround, delete the `DataVolume` object that is requesting the clone. When the source is available, recreate the `DataVolume` object that you deleted so that the cloning operation can complete successfully.
+* You can now configure the hostpath provisioner (HPP) to provide storage for virtual machine disks with dedicated storage that is separate from the node's root filesystem.  Otherwise, the node might run out of storage and become non-functional. A new `HPPSharingPoolPathWithOS` alert also now warns administrators if this misconfiguration occurs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038985[*BZ#2038985*])
 
-* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885605[*BZ#1885605*])
+* If you provision a virtual machine disk, {VirtProductName} now allocates a PVC that is just large enough to accommodate the requested disk size, rather than a `KubePersistentVolumeFillingUp` alert occurring for each VM disk PVC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039489[*BZ#2039489*])
 
-** As a workaround, you can use a secondary network interface connected to your host, or switch to the OpenShift SDN default CNI provider.
+* If you create a persistent hot-plug volume and a VMSnapshot, now you also create a VolumeSnapshot instance for each volume in the VMSnapshot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2042908[*BZ#2042908*])
```

Comment:
Suggest rephrasing this as:

"You can now create a virtual machine snapshot for VMs with hot-plugged disks."